modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayRadio, ClayRadioGroup} from '@clayui/form';
+import React, {ReactText} from 'react';
+
+import {FieldBase} from '../FieldBase';
+
+interface RadioProps {
+	defaultValue?: string;
+	disabled?: boolean;
+	id?: string;
+	inline?: boolean;
+	label?: string;
+	onChange: (value: ReactText) => void;
+	options: {
+		label: string;
+		value: string;
+	}[];
+	popover?: {
+		alignPosition?: 'top' | 'bottom';
+		content?: string;
+		disableScroll?: boolean;
+		header?: string;
+	};
+	required?: boolean;
+	tooltip?: string;
+}
+
+export function Radio({
```

Comment:
@danvb112 shouldn't we call it RadioButton/RadioField? I'm not sure if Radio is enough to describe this component.

Reply:
Thanks for the sugestion! I put the folder name and component name as RadioField to follow React Conventions